static/app/views/replays/replayTable.tsx

```diff
@@ -25,19 +26,28 @@ import useProjects from 'sentry/utils/useProjects';
 import {useRoutes} from 'sentry/utils/useRoutes';
 import type {ReplayListLocationQuery, ReplayListRecord} from 'sentry/views/replays/types';
 
+type ReplayRow = ReplayListRecord & {
+  txEvent?: {[x: string]: any};
```

Comment:
This could be something that `useReplaysFromTransaction.tsx` defines. The purpose of that file is really to do the query, and produce this list of objects.